doc/articles/getting-started/wizard/extensions.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: Uno.GettingStarted.UsingWizard.Extensions
+---
+
+Enables setting which Uno Extensions should be included with the generated projects.
+Uno Extensions is a set of Extensions that cover common essentials application core building blocks, and are supported on all operating systems Uno Platform targets.  
+The Uno.Extensions follows the Microsoft.Extensions model that creates a host environment where you can register additional dependencies. The registered dependencies are then available throughout the application via the Services (`IServiceProvider`) property on the `IHost` instance.
+    
+To learn more about Uno Extensions, visit [this page](xref:Overview.Features).
+    
+The available extensions are:
+    
+- **General**
+  - **Dependency Injection**
+    Dependency Injection (DI) is an Inversion of Control (IoC) technique which enables consuming classes not to worry about the management or creation of their dependencies, and instead rely on a central service to provide it when needed, based how the provider was configured.      
+    Uno Platform brings the power of [Microsoft Extensions Dependency Injection](https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection) to all OSs Uno Platform targets, and adds its additional functionality via its Uno.Extensions.DependencyInjection library, which you can learn more about [here](xref:Overview.DependencyInjection).
+
```

Comment:
Make a Note on Dependency Injection that some of the other extensions and options require this option to be enabled